mk/uvmt/vcs.mk

```diff
@@ -52,6 +52,7 @@ VCS_UVM_VERBOSITY ?= UVM_MEDIUM
 #VCS_UVMHOME_ARG ?= /opt/uvm/1800.2-2017-0.9/
 #VCS_UVMHOME_ARG ?= /opt/synopsys/vcs-mx/O-2018.09-SP1-1/etc/uvm
 VCS_UVMHOME_ARG  ?= /synopsys/vcs/S-2021.09-SP1/etc/uvm
+export VCS_HOME  ?= $(abspath $(shell which $(VCS))/../../)
```

Comment:
Will this work reliably for every flavour of Linux?

Reply:
Note that this is how it has been done for `UVM_HOME` for 4 years:
https://github.com/openhwgroup/core-v-verif/blame/0036e69b4d37388c86b3138ca2e31d0e889e11f7/mk/uvmt/vsim.mk#L36
`UVM_HOME               ?= $(abspath $(shell which $(VLIB))/../../verilog_src/uvm-1.2/src)`

Reply:
With that in mind, I think we could trust this way of doing it.
It is anyway better to have this than today's current failure that simply appears completely unrelated.
I would vote for Mario's fix, and rather change it to the "user must specify this env var" _if_ someone comes along and reports problems.
4 years of this way of doing it suggests that it is quite robust.

Reply:
Yes, this way has been the way of the Makefiles all this time

Reply:
Alrighty then.   Let's merge it as-is.